### PR TITLE
Code party

### PR DESCRIPTION
**Pull request by andeeliao**
Event page for code party

### CODE REVIEW

**Comment by ehashman** [comment]
Please rebase your commits, you have extraneous merges.

File: content/pages/exec-S15.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```
Comment (ehashman) [inline content/pages/exec-S15.md:1]: You have a merge conflict here that you're trying to commit :(

File: content/F2015/F15-event-code-party.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+title: Code Party 
+Date: 2015-11-17 17:00
+Category: Events
+Tags: codeparty
```
Comment (ehashman) [inline content/F2015/F15-event-code-party.md:4]: You can put a space in "code party" and also add "social" and "open source".

Comment (ehashman) [inline content/F2015/F15-event-code-party.md:4]: ...or maybe given the description, s/open source/academic/

**Comment by ehashman** [comment]
Less the comments, LGTM. Will need to fix the git problems before this is mergeable though.

**Comment by ehashman** [comment]
Okay, if you insist on closing this pull request, please run the following commands (assuming you've set up your repo like I suggested in the README):

```
git checkout master
git fetch upstream
git rebase upstream/master
git checkout -b code-party-secondattempt
git cherry-pick 80b5d49
git push origin HEAD
```

**Comment by andeeliao** [comment]
Hey Elana, 

I tried those commands but am still having the same problem. It seems that when I fetch upstream, rebase and checkout my master branch still has the extraneous commits... not sure how to fix this :S the secondattempt still has the commits from september

**Comment by fboxwala** [comment]
Hey Andee, wanna try this instead:

git reset --hard 118ebcb
git fetch upstream
git rebase upstream/master
git push --force origin HEAD

**Comment by ehashman** [comment]
+1 to Fatema's instructions. I didn't realize that your master itself had the merge problem, which means you'll need to reset that in addition to this branch!

**Comment by andeeliao** [comment]
Hmm, it still shows that I'm 1 commit ahead on my master branch. I checkout'd master before I applied Fatema's instructions, not sure if that makes a difference.

**Comment by ehashman** [comment]
Please hop on IRC so we can pair on this!

**Comment by andeeliao** [comment]
Roger that!

**Pull request closed without merging**